ProboCI/probo-asset-receiver
# Feature/addition rest endpoints

Connects #7 

Builds off of https://github.com/ProboCI/probo-asset-receiver/pull/6 https://github.com/ProboCI/probo-asset-receiver/pull/8 and https://github.com/ProboCI/probo-asset-receiver/pull/9

Adds additional API endpoints needed to allow users to review the size of their assets and remove them if they so desire.

NOTE: Only the last 5 commits are part of this PR.

### To Test:
 - You will first need to run the migrations (see testing instructions https://github.com/ProboCI/probo-asset-receiver/pull/6)
 - Upload new files
 - View assets in your bucket at `/buckets/:bucketName/assets`
 - Ensure the sizes and other attributes are appropriate for the file you uploaded.
 - Delete the file via the new endpoint `/buckets/:bucketName/assets/:assetName`
 - Ensure that the db keys and the files on disk are deleted.
